spec/fixtures/odk/forms/conditional_logic.xml

```diff
@@ -151,7 +158,8 @@
       <bind nodeset="/data/*itemcode7*_2" type="select1" relevant="((/data/*itemcode3* &gt; 5) and (/data/*itemcode6* = '*optcode1*')) and (not((/data/*itemcode3* = 0)))"/>
       <bind nodeset="/data/*itemcode8*" type="select"/>
       <bind nodeset="/data/*itemcode9*" type="dateTime"/>
-      <bind nodeset="/data/*itemcode10*" type="int" constraint="(. &gt;= 10 and . &lt;= 100) and ((/data/*itemcode4* = 10) or (. = /data/*itemcode3*)) and ((. != 55))" jr:constraintMsg="Valid only if: Must be greater than or equal to 10 and less than or equal to 100; [DecimalQ4] is equal to 10 OR This question is equal to [IntegerQ3]; This question is not equal to 55"/>
+      <bind nodeset="/data/*itemcode10*" type="int" constraint="(. &gt;= 10 and . &lt;= 100) and ((/data/*itemcode4* = 10) or (. = /data/*itemcode3*)) and ((. != 55))" jr:constraintMsg="Must be greater than or equal to 10 and less than or equal to 100; Valid only if: [DecimalQ4] is equal to 10 OR This question is equal to [IntegerQ3]; This question is not equal to 55"/>
```

Comment:
This is not quite the right expectation. I would like to see:

`Must be greater than or equal to 10 and less than or equal to 100; Valid only if: [DecimalQ4] is equal to 10 OR This question is equal to [IntegerQ3]; Valid only if this question is not equal to 55`

Reply:
Note the extra 'valid only if` in the last clause. Without that, it doesn't make sense.

Reply:
Ah, yeah, makes sense.